main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/DebuggingService.cs

```diff
@@ -905,6 +905,23 @@ static async void OnBusyStateChanged (object s, BusyStateEventArgs args)
 			await Runtime.RunInMainThread (delegate {
 				busyEvaluator.UpdateBusyState (args);
 				if (args.IsBusy) {
+					var session = (DebuggerSession) s;
+
+					if (sessions.TryGetValue (session, out var manager)) {
+						var metadata = new Dictionary<string, object> {
+							["DebuggerType"] = manager.Engine.Id,
+							["Debugger.AsyncOperation.Description"] = args.Description,
+							["Debugger.EvaluationOptions.AllowDisplayStringEvaluation"] = args.EvaluationContext.Options.AllowDisplayStringEvaluation,
+							["Debugger.EvaluationOptions.AllowMethodEvaluation"] = args.EvaluationContext.Options.AllowMethodEvaluation,
+							["Debugger.EvaluationOptions.AllowTargetInvoke"] = args.EvaluationContext.Options.AllowTargetInvoke,
+							["Debugger.EvaluationOptions.AllowToString"] = args.EvaluationContext.Options.AllowToStringCalls,
```

Comment:
```suggestion
							["Debugger.EvaluationOptions. AllowToStringCalls"] = args.EvaluationContext.Options.AllowToStringCalls,
```

If we want to mirror the property name. The rest of them do, seemed inconsistent